doc/source/user_guide/style.ipynb

```diff
@@ -47,6 +47,7 @@
    "outputs": [],
    "source": [
     "import matplotlib.pyplot\n",
+    "\n",
```

Comment:
why did this file change?

Reply:
Happened after I formatted using black running:
 ```bash
black .
```
in bash.
It seems like it would be better if I put the reasons behind my changes in the description. Would do that next time.

Reply:
no worries - let's revert this, CI isn't running `black-jupyter`

Reply:
if you run black via `pre-commit run black --all-files` then that'll match what's in CI